### PR TITLE
added recommendation

### LINKED ISSUE #34: Create recipe recommendation system

### LINKED ISSUE #35: Create api call to get userid from adatabase of logged-in user in auth0

### LINKED ISSUE #36: Unit test recommendation system

### PR DESCRIPTION
**Pull request by HamzaKazmi**
1) Created recommendation functionality under the Recommendation.js component.
2) Created API calls in server.js to get recommendations for the logged-in user
3) Worked with Nayab to create API functionality to get user-id

### CODE REVIEW

**Comment by defnecelebi** [comment]
Recommendation functionality is an important feature to have in our application and you handled it very good! This functionality requires high level of knowledge on Machine Learning and your implementation shows that you knew what you were working on. Overall, your code looks clean, readable and easy to understand. Keep up the good work!

File: client/src/components/Recommendation.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import { useAuth0 } from "@auth0/auth0-react";
+import React, { useState, useEffect } from "react";
+import JSONPretty from 'react-json-pretty';
+
+
+//Recommendation component
+const Recommendations = () => {
```
Comment (nb03) [inline client/src/components/Recommendation.js:7]: looks good!

Comment (nb03) [reply client/src/components/Recommendation.js:7]: has the function passed testing?

**Review by nb03 — COMMENTED** [review]
Great job Hamza! Looks like you added a function to handle the case when user has not liked a recipe and needs a recommendation. Have you added error handling?

File: client/src/components/Recommendation.js (hunk 1 above)
Comment (HamzaKazmi) [reply client/src/components/Recommendation.js:7]: Thank you for the feedback Nayab, I added a error handler that returns error message if the method fails despite all the steps taken to ensure it does not.

**Pull request closed without merging**